travis/apis.py

```diff
@@ -54,6 +56,19 @@ def __init__(self):
         self.ssh = os.environ.get(
             "WEBLATE_SSH", "ssh://user@webpage.com")
         self.tempdir = os.path.join(tempfile.gettempdir(), 'weblate_api')
+        self._ssh_keyscan()
+
+    def _ssh_keyscan(self):
+        """This method execute the command 'ssh-keysan' to avoid the
+        question when the command git clone is excecuted.
+        The question is like to:
+            'Are you sure you want to continue connecting (yes/no)?'"""
+        cmd = ['ssh-keyscan', '-p']
+        match = re.search(r'@([^/]+(?=:|/]))?(?::)?([^/]+)?/', self.ssh)
+        if match:
```

Comment:
`if not match: return False`
Return early

Reply:
Sorry for forget this